src/from_codec2/kiss_fft.c

```diff
@@ -12,7 +12,7 @@ Redistribution and use in source and binary forms, with or without modification,
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-
+#include "codec2_renames.h"
```

Comment:
Unfortunately we'll have to remember to re-add this every time these files get updated. Supposedly we can conditionally make these hidden (see [this](https://gcc.gnu.org/wiki/Visibility)) but that change may be better done in Codec2.

Reply:
I think that's OK.  Hmm, I wonder if it's possible to include a file on the gcc command line, much like we can -D a define :thinking:

Reply:
How about `-include` https://linux.die.net/man/1/gcc ?

Reply:
That works. Just pushed a new commit using this.

Reply:
Cool.  It's quite a neat solution now :+1: